tools/java/org/unicode/cldr/util/SupplementalDataInfo.java

```diff
@@ -4494,14 +4494,17 @@ public UnitPreferences getUnitPreferences() {
 
     /**
      * Grammar info for locales, with inheritance
+     * @param seedOnly TODO
      */
-    public GrammarInfo getGrammarInfo(String locale) {
-        while (locale != null) {
+    public GrammarInfo getGrammarInfo(String locale, boolean seedOnly) {
```

Comment:
Maybe use an enum over a boolean here (readability).

supplementalData.getGrammarInfo(cldrFile.getLocaleID(), ALL_LOCALES)
seems to read better than:
supplementalData.getGrammarInfo(cldrFile.getLocaleID(), false)

Reply:
This is temporary, since we'll be removing SEED next release. Did make a few other changes for simplicity